lang/ui.ca.json

```diff
@@ -0,0 +1,1161 @@
+    "description": "Ideas tab description text. Tab shows programs to give students ideas."
+  },
+  "insert-code-action": {
+    "defaultMessage": "Insereix codi"
+  },
+  "language": {
+    "defaultMessage": "Idioma",
+    "description": "Language option text"
+  },
+  "less-action": {
+    "defaultMessage": "Menys",
+    "description": "Less button text (showing less content)"
+  },
+  "load-error-makecode-info": {
+    "defaultMessage": "Aquest fitxer hexadecimal no es pot carregar a l'Editor de Python. L'Editor Python no pot obrir fitxers hexadecimals creats amb Microsoft MakeCode.",
+    "description": "Load error message"
+  },
+  "load-error-makecode-link": {
+    "defaultMessage": "Visita <link>https://makecode.microbit.org/</link> per utilitzar aquest fitxer hexadecimal.",
+    "description": "Load error message"
+  },
+  "load-error-mixed": {
+    "defaultMessage": "Un fitxer hexadecimal només es pot carregar sol. Substitueix tots els fitxers del projecte.",
+    "description": "Load error message"
+  },
+  "load-error-mpy": {
+    "defaultMessage": "Aquesta versió de l'Editor de Python actualment no admet l'addició de fitxers .mpy.",
+    "description": "Load error message"
+  },
+  "load-error-title": {
+    "defaultMessage": "No es pot carregar {fileCount, plural, one {file} other {files}}",
+    "description": "Title of error dialog when we cannot load files. Uses ICU syntax for pluralisation: https://formatjs.io/docs/core-concepts/icu-syntax/#plural-format."
+  },
+  "loaded-file-feedback": {
+    "defaultMessage": "S'ha carregat {filename}",
+    "description": "Success feedback for loading file"
+  },
+  "loading": {
+    "defaultMessage": "S'està carregant…",
+    "description": "Shown on loading indicators"
+  },
+  "microbit-hearts-alt": {
+    "defaultMessage": "placa micro:bit amb la graella LED de 5 per 5 que mostra un cor",
+    "description": "Alt text for micro:bit image in About dialog"
+  },
+  "micropython-documentation": {
+    "defaultMessage": "Documentació de MicroPython",
+    "description": "MicroPython documentation menu option text"
+  },
+  "micropython-history": {
+    "defaultMessage": "Descobreix com va néixer MicroPython a la micro:bit",
+    "description": "Links to MicroPython history in the about dialog"
+  },
+  "micropython-source-code": {
+    "defaultMessage": "MicroPython <linkV1>source code for the micro:bit V1</linkV1>(codi font per la micro:bit V1) i <linkV2>micro:bit V2</linkV2>",
+    "description": "Links to the MicroPython source code in the about dialog"
+  },
+  "more-action": {
+    "defaultMessage": "Més",
+    "description": "More button text (showing more/further content)"
+  },
+  "more-connect-options": {
+    "defaultMessage": "Més opcions de connexió",
+    "description": "Aria label for the additional actions menu to the right of the Send to micro:bit button"
+  },
+  "more-ideas": {
+    "defaultMessage": "<link>More ideas</link> Més idees.",
```

Comment:
This should probably be <link>Més idees</link> with no period.

Reply:
Changed in Crowdin